<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterWindow</class>
 <widget class="QMainWindow" name="EnterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string>
     background-color: #f5f5f5;
     font-family: Arial, sans-serif;
    </string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome Back</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      color: #333;
      padding: 10px;
      border-bottom: 2px solid #aaa;
    </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your login</string>
    </property>
    <property name="styleSheet">
     <string>
      border: 1px solid #aaa;
      border-radius: 5px;
      padding: 5px;
      background-color: #fff;
    </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your password</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="styleSheet">
     <string>
      border: 1px solid #aaa;
      border-radius: 5px;
      padding: 5px;
      background-color: #fff;
    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
    <property name="styleSheet">
     <string>
      background-color: #007BFF;
      color: #fff;
      border: none;
      border-radius: 5px;
      padding: 10px;
    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
    <property name="styleSheet">
     <string>
      background-color: #28a745;
      color: #fff;
      border: none;
      border-radius: 5px;
      padding: 10px;
    </string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string>
      color: red;
    </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>


